src/app/screens/BookDetail/components/BookCard/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import i18next from 'i18next';
+import { useHistory } from 'react-router';
+
+import { BookSuccess } from '~utils/types';
+
+import BackArrow from '../../Assets/back-arrow.png';
+import Badge from '../../Assets/badge.png';
+
+import styles from './styles.module.scss';
+
+interface BookProps {
+  data?: BookSuccess;
```

Comment:
this shouldnt be optional, you only render a book if you have its complete information. Or if you render it if it doesnt have any data, you should catch this case inside the component and render some error